kf_lib_data_ingest/etl/load/load.py

```diff
@@ -417,53 +427,53 @@ def _load_entity(self, entity_type, endpoint, entity_id, body, links):
 
         # log action
         with count_lock:
+            msg_cache = os.path.join(self.stage_cache_dir, "SentMessages.txt")
```

Comment:
We already talked about this in planning but reiterating. Can we make this a JSON file and have each payload include the message body and URL